browser/ui/views/commands/default_accelerators_mac.mm

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/browser/ui/views/commands/default_accelerators_mac.h"
+
+#import <Cocoa/Cocoa.h>
+
+#include "base/check.h"
+#include "base/containers/contains.h"
+#include "base/containers/flat_map.h"
+#include "base/logging.h"
+#include "base/strings/sys_string_conversions.h"
+#include "brave/app/command_utils.h"
+#include "chrome/app/chrome_command_ids.h"
+#include "chrome/browser/global_keyboard_shortcuts_mac.h"
+#include "chrome/browser/ui/cocoa/accelerators_cocoa.h"
+#include "ui/events/cocoa/cocoa_event_utils.h"
+#include "ui/events/event_constants.h"
+#include "ui/events/keycodes/keyboard_code_conversion_mac.h"
+
+static_assert(__OBJC__);
+static_assert(BUILDFLAG(IS_MAC));
+
+namespace commands {
+
+namespace {
+
+bool CanConvertToAcceleratorMapping(int command_id) {
+  return base::Contains(commands::GetCommands(), command_id);
```

Comment:
Are there commands which fail this?

Reply:
Yeah, there are a bunch of shortcuts that fail this. Let's see..

```

// For editing contents
CanConvertToAcceleratorMapping not contianed 50140   // IDC_CONTENT_CONTEXT_COPY 
CanConvertToAcceleratorMapping not contianed 50141   // IDC_CONTENT_CONTEXT_CUT
CanConvertToAcceleratorMapping not contianed 50142   // IDC_CONTENT_CONTEXT_PASTE 
CanConvertToAcceleratorMapping not contianed 50143   // IDC_CONTENT_CONTEXT_DELETE
CanConvertToAcceleratorMapping not contianed 50144   // IDC_CONTENT_CONTEXT_UNDO
CanConvertToAcceleratorMapping not contianed 50145   // IDC_CONTENT_CONTEXT_REDO
CanConvertToAcceleratorMapping not contianed 50146   // IDC_CONTENT_CONTEXT_SELECTALL 
CanConvertToAcceleratorMapping not contianed 50147   // IDC_CONTENT_CONTEXT_PASTE_AND_MATCH_STYLE 
CanConvertToAcceleratorMapping not contianed 50158   // IDC_CONTENT_CONTEXT_SPELLING_TOGGLE

// Spell checker
CanConvertToAcceleratorMapping not contianed 41005   // IDC_SPELLCHECK_MENU   
CanConvertToAcceleratorMapping not contianed 40003   // IDC_SPELLCHECK_SUGGESTION_3 

CanConvertToAcceleratorMapping not contianed 40029  // IDC_BOOKMARKS_MENU 

// In multi selected tabs context
CanConvertToAcceleratorMapping not contianed 35050  // IDC_MUTE_TARGET_SITE  
CanConvertToAcceleratorMapping not contianed 35051  // IDC_PIN_TARGET_TAB   
CanConvertToAcceleratorMapping not contianed 35052  // IDC_GROUP_TARGET_TAB     
CanConvertToAcceleratorMapping not contianed 35053  // IDC_DUPLICATE_TARGET_TAB 

CanConvertToAcceleratorMapping not contianed 400    ??
CanConvertToAcceleratorMapping not contianed 401    ??
CanConvertToAcceleratorMapping not contianed 440    ??
CanConvertToAcceleratorMapping not contianed 441    ??
CanConvertToAcceleratorMapping not contianed 480    ??

```

Reply:
Aah right, that make sense.

1. `CONTENT_CONTEXT` commands are only available from the content context menu
2. The `TARGET` commands are only available from the Right Click menu on tabs. 

Sweet, thanks forn checking that @sangwoo108